lib/ansible/plugins/callback/__init__.py

```diff
@@ -555,11 +555,32 @@ def v2_runner_on_skipped(self, result: TaskResult) -> None:
             host = result._host.get_name()
             self.runner_on_skipped(host, self._get_item_label(getattr(result._result, 'results', {})))
 
-    def v2_runner_on_unreachable(self, result):
+    def v2_runner_on_unreachable(self, result: TaskResult) -> None:
+        """Get details about a task that Ansible could not perform because it could not reach a
+        targeted host, then process the details as required by the callback (output, profiling,
+        logging, notifications, etc.)
+
+        Customization note: For more information about the attributes and methods of the
+        TaskResult class, see lib/ansible/executor/task_result.py.
+
+        :param TaskResult result: An object that contains details about the task
```

Comment:
I find it easier to consume this syntax when type is on a separate line:
```suggestion
        :param result: An object that contains details about the task
        :type result: TaskResult
```

Reply:
Done, but I had to use a continuation character for the string to appear correctly in PyCharm.

Reply:
Oh, interesting.. Sometimes it does require extra empty lines and other tricks like turning the docstring into an r-docstring. It depends on the content inside.

Anyway, I'm marking this thread as resolved.